[FIX] web: clickbot: do not throw is no app to test

aab-odoo opened the pull request:
Commit [1] slightly reworked the clickbot w.r.t. the Dropdown refactoring (especially for menus being dropdowns). It introduced a throw Error if it detects no app. The goal was to indicate that something might be wrong with the code of the clickbot (i.e. there are apps, but it didn't find them because selectors are broken).

However, we run the clickbot in various situations (single apps, admin/demo...), and it happens that there is actually no app (see [2]).

We thus remove that error. There's a qunit suite for the clickbot anyway, so this is tested.

[1] 384c992ff09f6bd3673563a26e75b3d1aa41d8b3
[2] https://runbot.odoo.com/web/#id=57211&view_type=form&model=runbot.build.error&menu_id=405&cids=1

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/155530).

robodoo commented:
Merge method set to rebase and fast-forward.

aab-odoo commented:
Note:
Added the first time by https://github.com/odoo/odoo/pull/146174
Removed by https://github.com/odoo/odoo/pull/149044
Re-introduced by https://github.com/odoo/odoo/pull/137691, probably when rebasing, inadvertently 

@jpp-odoo
